crates/data_components/src/util/indexes.rs

```diff
@@ -44,38 +41,10 @@ impl Display for IndexType {
     }
 }
 
-#[must_use]
-pub fn indexes_from_option_string(indexes_option_str: &str) -> HashMap<String, IndexType> {
-    indexes_option_str
-        .split(';')
-        .map(|index| {
-            let parts: Vec<&str> = index.split(':').collect();
-            if parts.len() == 2 {
-                (parts[0].to_string(), IndexType::from(parts[1]))
-            } else {
-                (index.to_string(), IndexType::Enabled)
-            }
-        })
-        .collect()
-}
-
-pub fn index_columns(indexes_key: &str) -> Vec<&str> {
```

Comment:
This used to be the parsing logic for the column references before I moved it to be in its own struct in this PR.